### ISSUE TITLE
[Screen Reader-Header controls-Projects]Narrator is not providing the correct information about the ‘Project’ Menu items showing under header Controls.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2564]
**Pre-Requisite:** Turn on Narrator(Win + Ctrl + Enter)

**User Experience:** Users who depends on Screen Reader will get confused if Narrator will not provide correct information about the Menu items.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Tool used:-IE Developer (F12)

**Repro Steps:-**
1. Navigate to https://makecode.microbit.org/acc
2. Navigate to the Microbit section element and select Code control given on it.
3. Navigate to the Projects control lying in the header section on the page.
6. Verify that Narrator is informing correct information about the 'Project' Menu item or not.
 
**Actual Result:-**
Narrator is not providing the correct information about the ‘Project’ Menu items showing under header Controls as it read the title and aria label both and they defined as same.

**Expected Result:-**
Narrator should provide correct information about the ‘Project’ Menu items showing under header Controls ,It should not read same values for title , aria label both OR different values should be defined for title and aria-label (Could be more descriptive) 

**Note:** This same issue does not exists with NVDA + FF

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

**Please refer the attachment.**
![mas4 2 1_header control_project](https://user-images.githubusercontent.com/30459830/28705689-a02543c0-738e-11e7-97b7-8c969433147c.jpg)

[MAS4.2.1_Header.Control_Project.zip](https://github.com/Microsoft/pxt/files/1192961/MAS4.2.1_Header.Control_Project.zip)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/acc and issue is fixed now. Hence closing the bug.
